src/index.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
```

Comment:
This file seems to be doing 3-4 different things - it's defining the various bits of the app, it's building the view, it's defining some behaviour to the view, and it's then applying the view to the page. 

In your self-review you mentioned: "I have been slow in implementing the code for calling the code onto the html page and would like to focus on building an effective means of doing this." I think that fact that you haven't broken it down a bit more has probably contributed to this - an approach you might try in future:

1) Write the HTML, with dummy data and no behaviour
2) Write some code for adding the code to the page
3) Extract the data into a model - e.g. title, content, image url
4) Write some code to bind data to the view
5) Extract behaviour into your model e.g. callbacks
6) Write some code to bind callbacks to the view

Approaching it in this fashion should help you separate concerns, focus on one thing at a time, and maintain a visual measure of progress.